astroid/brain/brain_namedtuple_enum.py

```diff
@@ -157,10 +157,14 @@ def infer_func_form(
     # we know it is a namedtuple anyway.
     name = name or "Uninferable"
     # we want to return a Class node instance with proper attributes set
-    class_node = nodes.ClassDef(name, "docstring")
-    class_node.parent = node.parent
-    # set base class=tuple
-    class_node.bases.append(base_type)
+    class_node = nodes.ClassDef(name, parent=node.parent)
+    class_node.postinit(
+        # set base class=tuple
+        bases=[base_type],
+        body=[],
+        decorators=None,
+        doc_node=nodes.Const(value="docstring"),
```

Comment:
Same here.

Reply:
This sets the base class, so I guess this is doing more than only set the docstring?

Reply:
Only remove the `doc_node` attribute here